core/src/main/java/com/avast/bytes/Bytes.java

```diff
@@ -82,12 +102,19 @@ default boolean isEmpty() {
 
     /**
      * Converts content of this {@link Bytes} to hex string. It uses data provided by `toByteArray`, which means it probably
-     * create a copy of the data (but that depends on the implementation of the `toByteArray`.
+     * creates a copy of the data (but that depends on the implementation of the `toByteArray`).
      *
+     * Note: Implementation was copied from javax.xml.bind package because the package is no longer supported in jdk11
      * @return hex string representation
      */
     default String toHexString() {
-        return DatatypeConverter.printHexBinary(toByteArray()).toLowerCase();
+        byte[] data = toByteArray();
+        StringBuilder r = new StringBuilder(data.length * 2);
+        for (byte b : data) {
+            r.append(Hidden.hexCode[(b >> 4) & 0xF]);
+            r.append(Hidden.hexCode[(b & 0xF)]);
+        }
+        return r.toString().toLowerCase();
```

Comment:
If you want lower case, change the characters in `hexCode`, this doesn't make sense.